src/modules/products/products.service.ts

```
@@ -0,0 +1,312 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { RequestContextService } from 'src/common/context/request-context/request-context.service';
+import { ErrorCode } from 'src/common/errors/error-codes.enum';
+import { throwHttpError } from 'src/common/errors/http-exception.helper';
+import { Category } from 'src/entities/category.entity';
+import { Product } from 'src/entities/product.entity';
+import { Store } from 'src/entities/store.entity';
+import { MyLoggerService } from 'src/my-logger/my-logger.service';
+import { Repository, FindOptionsWhere, Not, ILike } from 'typeorm';
+import * as crypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)